### DISCUSSION TITLE
Navigation para mudar de telas

### DISCUSSION THREAD

**Discussion by LuccasHuzuniV3** [Flutterando/forum#31] [Perguntas e Respostas] [answered]
salve galera boa noite tudo firmeza ?
galera sou iniciante e estou com um pequeno problema ao tentar mudar de telas usando o Navigator.
![image](https://user-images.githubusercontent.com/103328071/190953868-b5649234-3cc0-4f30-a59f-34f356cde663.png)
Eu nao sei oque devo fazer

**Comment by eduardoflorence** [comment, marked as answer]
Você provavelmente fez um import da lib dart:js por engano, é só eliminar esse import. Essa lib também possui o context e por isso o conflito
